src/app/public/modules/data-manager/data-manager.service.ts

```diff
@@ -257,9 +257,9 @@ export class SkyDataManagerService implements OnDestroy {
   }
 
   /**
-   * Update the given view config. The registered view with the same ID is updated to the provided config,
-   * so all properties (changed or unchanged) should be included. If the view has not been initialized already no update will happen.
-   * @param view The new SkyDataViewConfig containing all properties.
+   * Updates the given view config. The registered view with the same ID is updated to the provided config,
+   * so all properties (changed or unchanged) are included. If the view was not initialized already, no update happens.
```

Comment:
This is trying to say all properties (changed or unchanged) should be included by the consumer when they pass the param, so I don't think "are" is the right word choice here

Reply:
Would `... so include all properties regardless of whether they changed.` work? ... I'm going to go ahead and push that change up, but just let me know if it doesn't work.